docs/reference/mapping/runtime.asciidoc

```diff
@@ -193,48 +190,164 @@ script, and returns the value as part of the query. Because the runtime field
 shadows the mapped field, you can modify the value returned in search without
 modifying the mapped field.
 
-For example, index the following documents into `my-index`:
+For example, let's say you indexed the following documents into `my-index`:
 
-[source, console]
+[source,console]
 ----
 POST my-index/_bulk?refresh=true
 {"index":{}}
-{"name":{"first":"Jose","last":"Hickman"}}
+{"timestamp":1516729294000,"model_number":"QVKC92Q","measures":{"voltage":5.2}}
+{"index":{}}
+{"timestamp":1516642894000,"model_number":"PW83H7X","measures":{"voltage":5.8}}
+{"index":{}}
+{"timestamp":1516556494000,"model_number":"ELF7YV2","measures":{"voltage":5.1}}
+{"index":{}}
+{"timestamp":1516470094000,"model_number":"5NMDTMF","measures":{"voltage":5.6}}
 {"index":{}}
-{"name":{"first":"India","last":"Avila","suffix":"II"}}
+{"timestamp":1516383694000,"model_number":"1N0TH44","measures":{"voltage":4.2}}
+{"index":{}}
+{"timestamp":1516297294000,"model_number":"HG537PU","measures":{"voltage":4.0}}
 ----
 
-In the `runtime_mappings` section of the `_search` request, you can define a
-script that operates on the `name.first` field. The values you specify will
-shadow field values in the index mapping for the duration of the query without
-modifying the field itself.
+You later realize that the voltage for the sensor matching model number
+`HG537PU` is incorrect. The indexed value is `4.0`, but is supposed to be 1.7
```

Comment:
I'd say something like "Those darned HG537PUs report 90% of the real voltage" or something like that. Sounds kind of believable.

Reply:
"Sir, I was told these were quality sensors."